src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -232,6 +232,14 @@ void java_bytecode_convert_method_lazy(
   method_symbol.mode=ID_java;
   method_symbol.location=m.source_location;
   method_symbol.location.set_function(method_identifier);
+  if(m.is_public)
+    member_type.set(ID_access, ID_public);
```

Comment:
Shouldn't this be using the same `set_access` API? (`member_type` will thus need to be converted to a more specific type than `typet`.)